state/spacesdiscovery.go

```diff
@@ -48,17 +52,23 @@ func (st *State) ReloadSpaces(environ environs.Environ) error {
 		if err != nil {
 			return errors.Trace(err)
 		}
-		return errors.Trace(st.SaveSubnetsFromProvider(subnets))
+		return errors.Trace(st.SaveSubnetsFromProvider(subnets, ""))
 	}
 }
 
+type fanPair struct {
```

Comment:
This type feels like it should live in the network package.

Reply:
Done